build.sbt

```diff
@@ -66,13 +68,15 @@ dockerBaseImage := "openjdk:8-jre-alpine"
 dockerEntrypoint := Seq(s"/opt/docker/bin/${name.value}")
 
 dockerCommands := dockerCommands.value.flatMap {
-  case cmd@Cmd("WORKDIR", _) => List(
+  case cmd @ Cmd("WORKDIR", _) => Seq(
     Cmd("WORKDIR", "/src")
   )
-  case cmd@(Cmd("ADD", _)) => List(
+  case cmd @ Cmd("ADD", _) => Seq(
     Cmd("RUN", s"adduser -u 2004 -D $dockerUser"),
     cmd,
     Cmd("RUN", "mv /opt/docker/docs /docs")
   )
   case other => List(other)
 }
+
+s3region := Region.EU_Ireland
```

Comment:
Maybe a little comment explaining why is this necessary?

Reply:
It's gone with new `codacy-sbt` update.